python/hsfs/expectation_suite.py

```diff
@@ -252,6 +257,30 @@ def add_expectation(
         """
         Append an expectation to the local suite or in the backend if attached to a Feature Group.
 
+        !!! example
+            ```python
+            expectation_suite.add_expectation(
+                ge.core.ExpectationConfiguration(
+                    expectation_type="expect_column_min_to_be_between",
+                    kwargs={
+                        "column": "foo_id",
+                        "min_value": 0,
+                        "max_value": 1
+                    }
+                )
+            )
+
+            expectation_suite.add_expectation(
+                ge.core.ExpectationConfiguration(
+                    expectation_type="expect_column_value_lengths_to_be_between",
+                    kwargs={
+                        "column": "bar_name",
+                        "min_value": 3,
+                        "max_value": 10
+                    }
+                )
+            )
+            ```
```

Comment:
is it possible to add short comment/explanation what are you doing with this snippet?